packages/Webkul/Admin/src/Resources/lang/it/app.php

```diff
@@ -1020,6 +1020,7 @@
             'choose-condition-to-add' => 'Scegli una condizione da aggiungere',
             'cart-attribute' => 'Attributi Carrello',
             'subtotal' => 'Subtotale',
+            'additional' => 'Zusatzinformationen',
```

Comment:
I thing this is not the right place for a german translation ;)

Reply:
Oww yeah, I took the wrong copy :O thanks for noticing me